applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -124,14 +124,29 @@ CONFIG_SLM_INDICATE_TIME - Indicate GPIO active time
    This option specifies the length, in milliseconds, of the time interval during which the indicate GPIO must stay active.
    The default value is 100 milliseconds.
 
-.. _CONFIG_SLM_SOCKET_RX_MAX:
+.. _CONFIG_SLM_AUTO_CONNECT:
 
-CONFIG_SLM_SOCKET_RX_MAX - Maximum RX buffer size for receiving socket data
-   This option specifies the maximum buffer size for receiving data through the socket interface.
-   By default, this size is set to :c:enumerator:`NET_IPV4_MTU` (576), which is defined in Zephyr.
-   The maximum value is 708, which is the maximum segment size (MSS) defined for the modem.
+CONFIG_SLM_AUTO_CONNECT - Connect to LTE network at start-up or reset
+   This option enables connecting to LTE network at start-up or reset using default PDN configuration.
+   This option is enabled by the LwM2M Carrier overlay, but is otherwise disabled by default.
 
-   This option impacts the total RAM usage.
+   .. note::
+      This option requires network-specific configuration in the ``slm_auto_connect.h`` file.
+
+   Sample configuration for NIDD connection in the :file:`slm_auto_connect.h` file::
```

Comment:
```suggestion
   Here is a sample configuration for NIDD connection in the :file:`slm_auto_connect.h` file::
```
Should this sample configuration be inside the note as well? If it is moved, you can remove the "in the :file:`slm_auto_connect.h` file" part.

Reply:
Let's keep it outside, have a look at [current layout](https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-12849/nrf/applications/serial_lte_modem/doc/slm_description.html#id10).